lib/web_ui/dev/integration_tests_manager.dart

```diff
@@ -71,12 +70,24 @@ class IntegrationTestsManager {
     }
   }
 
-  void _runDriver() async {
-    startProcess(
-      './chromedriver/chromedriver',
-      ['--port=4444'],
-      workingDirectory: io.Directory.current.path
+  Future<void> _cloneFlutterRepo() async {
+    final int exitCode = await runProcess(
+      environment.cloneScript.path,
+      <String>[
+        // Use .dart_tools to clone the Flutter repo.
+        environment.webUiDartToolDir.path,
```

Comment:
IIRC you said that we will also run e2e tests for non-Web tests. If that's the plan then we should find a better place to clone Flutter than `web_ui/.dart_tool` so we can share the same checkout instead of checking it out multiple times. Can we reuse the same directory that's used for existing tests today?

Reply:
I think we can. However all the non-web integration tests (as far as I know) is running on other LUCi bots and other cirrus tasks.

Let me know if I'm missing something.

Reply:
I only meant that we should have one flutter checkout that's used by all tests, Web, mobile, desktop, etc. If locally or on LUCI bot you run both Web e2e tests _and_ mobile e2e tests you will end up cloning the framework twice, which is wasteful and could lead to subtle bugs (e.g. some test can accidentally use the wrong checkout). So I suggest picking one standard location and cloning the framework there, e.g. `src/flutter/.dart_tool`.

Reply:
done.